### ISSUE TITLE
Errors on first run

### ISSUE THREAD

**Issue by nicwaller** [nicwaller/puppet-mailman#18]
On a new test system, encountered these errors on first run.

```
Error: Could not set 'directory' on ensure: Could not find group mailman at 37:/etc/puppet/modules/mailman/manifests/etclinks.pp
Error: Could not set 'directory' on ensure: Could not find group mailman at 37:/etc/puppet/modules/mailman/manifests/etclinks.pp
Wrapped exception:
Could not find group mailman
Error: /File[/etc/mailman]/ensure: change from absent to directory failed: Could not set 'directory' on ensure: Could not find group mailman at 37:/etc/puppet/modules/mailman/manifests/etclinks.pp
Notice: /File[/etc/mailman/virtual-mailman]: Dependency File[/etc/mailman] has failures: true
Warning: /File[/etc/mailman/virtual-mailman]: Skipping because of failed dependencies
```

and later

```
Notice: /File[/etc/mailman/aliases.db]: Dependency File[/etc/mailman] has failures: true
Warning: /File[/etc/mailman/aliases.db]: Skipping because of failed dependencies
Notice: /File[/etc/mailman/aliases]: Dependency File[/etc/mailman] has failures: true
Warning: /File[/etc/mailman/aliases]: Skipping because of failed dependencies
Notice: /File[/var/lib/mailman/data/adm.pw]/ensure: created
Notice: /File[/etc/mailman/adm.pw]: Dependency File[/etc/mailman] has failures: true
Warning: /File[/etc/mailman/adm.pw]: Skipping because of failed dependencies
Notice: /File[/etc/mailman/creator.pw]: Dependency File[/etc/mailman] has failures: true
Warning: /File[/etc/mailman/creator.pw]: Skipping because of failed dependencies
```

**Comment by nicwaller** [comment]
This may be resolved in e933c43 but requires more complete testing.

**Comment by nicwaller** [comment]
It gets farther now, but then runs into this:

```
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns: Traceback (most recent call last):
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:   File "/usr/lib/mailman/bin/newlist", line 260, in <module>
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:     main()
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:   File "/usr/lib/mailman/bin/newlist", line 202, in main
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:     emailhost=host_name, urlhost=urlhost)
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:   File "/usr/lib/mailman/Mailman/MailList.py", line 505, in Create
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:     self.InitVars(name, admin, crypted_password, urlhost=urlhost)
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:   File "/usr/lib/mailman/Mailman/MailList.py", line 409, in InitVars
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:     baseclass.InitVars(self)
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:   File "/usr/lib/mailman/Mailman/Archiver/Archiver.py", line 124, in InitVars
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:     }, mlist=self))
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:   File "/usr/lib/mailman/Mailman/Utils.py", line 587, in maketext
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:     return findtext(templatefile, dict, raw, lang, mlist)[0]
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:   File "/usr/lib/mailman/Mailman/Utils.py", line 564, in findtext
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns:     raise IOError(errno.ENOENT, 'No template file found', templatefile)
Notice: /Stage[main]/Mailman/Exec[create_site_list]/returns: IOError: [Errno 2] No template file found: 'emptyarchive.html'
Error: newlist -q 'mailman' 'mailman@localhost.home.nicwaller.com' 'CHANGEME' returned 1 instead of one of [0]
Error: /Stage[main]/Mailman/Exec[create_site_list]/returns: change from notrun to 0 failed: newlist -q 'mailman' 'mailman@localhost.home.nicwaller.com' 'CHANGEME' returned 1 instead of one of [0]
Notice: /Stage[main]/Mailman/Exec[change_site_list_pw]: Dependency Exec[create_site_list] has failures: true
Warning: /Stage[main]/Mailman/Exec[change_site_list_pw]: Skipping because of failed dependencies
Notice: /Stage[main]/Mailman/Service[mailman]: Dependency Exec[create_site_list] has failures: true
Warning: /Stage[main]/Mailman/Service[mailman]: Skipping because of failed dependencies
```

**Comment by nicwaller** [comment]
Appears to be resolved in eaa6a0f.
